LocalMultiplayerAgent/Program.cs

```diff
@@ -49,6 +48,14 @@ public static async Task Main(string[] args)
                 Environment.Exit(1);
             }
 
+            Globals.CreateDeployment = args.Contains("-deploy");
```

Comment:
shouldn't this code run after debugging? how do we know from that args parameters if users just want to run deployment script or run both of debugging and deployment script?

My understanding of this logic is when args include -deploy, we don't do debugging and just run deploymentscript?

Reply:
Yes, so if users want to run debugging before deployment, they run `.\LocalMultiplayerAgent.exe`. After this is ran, the program ends.
So to run deployment, the user will have to `.\LocalMultiplayerAgent.exe -deploy`, which runs the program again
They are two independent processes
Does that make sense?